test/old-tests/unit/test-kernel.cpp

```diff
@@ -2048,7 +2048,7 @@ GPU_TEST(Kernel, CudaExec1c)
 
   // Loop Fusion
   using Pol = KernelPolicy<
-      CudaKernelExt<cuda_explicit_launch<false, 5, 3>,
+      CudaKernelExt<cuda_explicit_launch<false, 5, 1, 3>,
```

Comment:
Shouldn't the extra 1 be at the end with the reordering?

Reply:
Good catch. I missed that.

Reply:
Thanks, I missed this too, I just fixed this in the fork PR.

Reply:
I fixed in here. Please double-check my change and approve if you think this is good to go. @corbett5 @MrBurmark is there an analogous change that should be made to the corresponding HIP policy?

Reply:
Thanks. I'm pretty sure there is. And also for flat `cuda_exec` it would be nice, but my use case at the moment involves kernel.

https://github.com/LLNL/RAJA/blob/dffd98b1ca3f4b0f8308fc0a78d1d66252250d47/include/RAJA/policy/hip/kernel/HipKernel.hpp#L193

Reply:
@corbett5 we will fill in the gaps soon, may not be in time for the next RAJA release, which is imminent. Also, I restarted the GItlab CI on corona. Not sure why it timed out without starting the builds.